test/common/testlib.py

```diff
@@ -528,6 +528,8 @@ def try_login(self, user, password, authorized=True):
         self.set_val('#login-user-input', user)
         self.set_val('#login-password-input', password)
         self.set_checked('#authorized-input', authorized)
```

Comment:
Should this then be only done for "distropkg"?

Reply:
Hmm, good point.  The "#authorized-input" now only affects SSH, and we could make that explicit in the tests.

My plan is to eventually remove `#authorized-input` completely. Would that be a better time to figure this out?

Reply:
We can clean it up in a follow-up for sure. I'd just like to have at least one explicit test case that makes sure that sudo works with that box unchecked.

Reply:
Indeed, that is a very good point!  I'll add that.

Reply:
Ok, I did this more properly after all.  login_and_go (et al) now have a "superuser" parameter (that is completely independent from "authorized") and the tests use that insted of "authorized". This means whatever we do about the #authorized-input should not affect most tests in the future.

I have also added a test with `login_and_go(authorized=False, superuser=True)`.